packages/mintlist-cli/src/cmd/gen-index.ts

```diff
@@ -0,0 +1,101 @@
+import { existsSync, mkdirSync, readdirSync, writeFileSync } from "mz/fs";
+import { resolve, relative } from "path";
+import { MintlistFileUtil } from "../util/mintlist-file-util";
+
+export function genIndex(opts: any) {
+  if (!isOpts(opts)) {
+    throw new Error("Invalid options");
+  }
+  const { outDir, inDir } = opts;
+  const inPath = resolve(inDir);
+  const outPath = resolve(outDir);
+  const files = getFilePaths(inPath);
+  if (files.length === 0) {
+    throw new Error(`No files found in ${inPath}`);
+  }
+
+  const contents = [
+    HEADER_LINE,
+    "",
+    ...files.map((file) => toImportLine(file, outPath)),
+    "",
+    ...files.map((file) => toExportLine(file)),
+    "",
+  ].join("\n");
+
+  if (!existsSync(outPath)) {
+    mkdirSync(outPath);
+  }
+
+  writeFileSync(`${outPath}/index.ts`, contents);
+}
+
+function toExportLine(path: string): string {
+  const fileName = path.split("/").pop();
+  if (!fileName) {
+    throw new Error("Invalid path");
+  }
+  return `export const ${toExportIdentifier(fileName)}: ${
+    isMintlistName(fileName) ? "Mintlist" : "Tokenlist"
```

Comment:
This is to distinguish between exporting a `mintlist.json` or a `tokenlist.json` so that the exported variable type is set properly - `export const MY_TOKEN_LIST: Tokenlist ...`